### ISSUE TITLE
Update Finnish translation in CLI

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-cli#264]
The Finnish translation (PO file) needs update for the upcoming release. Please follow the instructions in "[Instructions for translators]" or request for help from the Zonemaster project.

[Instructions for translators]: https://github.com/zonemaster/zonemaster/blob/develop/docs/internal-documentation/maintenance/Instructions-for-translators.md

Or provide the updated translations as comments to this issue. Messages that need translation or updated translation:
```po
msgid "Warning: Zonemaster::LDNS not compiled with IDN support, cannot handle non-"
"ASCII names correctly."
msgstr "Varoitus: Zonemaster::LDNS:n kääntämisessä ei ole käytetty libidn2:iä, ei "
"käsittele muussa kuin ASCII-muodossa olevia nimiä oikein."
```
```po
msgid "The domain name contains consecutive dots.\n"
msgstr ""
```
```po
msgid "--ns must be a name or a name/ip pair."
msgstr ""
```
```po
msgid "The name of the nameserver '{nsname}' contains consecutive dots."
msgstr ""
```

**Comment by samisalmensuo** [comment]
msgid "Warning: Zonemaster::LDNS not compiled with IDN support, cannot handle non-"
"ASCII names correctly."
msgstr "Varoitus: Zonemaster::LDNS ei ole käännetty IDN tuella, joten se ei pysty käsittelemään "
" Ei ASCII-muodossa olevia nimiä oikein."

msgid "The domain name contains consecutive dots.\n"
msgstr "Verkkotunnus sisältää peräkkäisiä pisteitä.\n"

msgid "--ns must be a name or a name/ip pair."
msgstr "--ns:n on oltava nimi tai nimi/ip-pari"

msgid "The name of the nameserver '{nsname}' contains consecutive dots."
msgstr "Nimipalvelimen nimi '{nsname}' sisältää peräkkäisiä pisteitä."

**Comment by samisalmensuo** [comment]
Hi,

I added translations as comments. I think I translated everything, but please let me know if anything is missing.


- Sami

-----Original Message-----
From: Mats Dufberg ***@***.***> 
Sent: sunnuntai 22. toukokuuta 2022 17.49
To: zonemaster/zonemaster-cli ***@***.***>
Cc: Salmensuo Sami ***@***.***>; Assign ***@***.***>
Subject: [zonemaster/zonemaster-cli] Update Finnish translation in CLI (Issue #264)

The Finnish translation (PO file) needs update for the upcoming release. Please follow the instructions in "Instructions for translators <https://github.com/zonemaster/zonemaster/blob/develop/docs/internal-documentation/maintenance/Instructions-for-translators.md> " or request for help from the Zonemaster project.

Or provide the updated translations as comments to this issue. Messages that need translation or updated translation:

msgid "Warning: Zonemaster::LDNS not compiled with IDN support, cannot handle non-"

"ASCII names correctly."

msgstr "Varoitus: Zonemaster::LDNS:n kääntämisessä ei ole käytetty libidn2:iä, ei "

"käsittele muussa kuin ASCII-muodossa olevia nimiä oikein."
msgid "The domain name contains consecutive dots.\n"

msgstr ""
msgid "--ns must be a name or a name/ip pair."

msgstr ""
msgid "The name of the nameserver '{nsname}' contains consecutive dots."

msgstr ""

**Comment by matsduf** [comment]
Thanks, I'll about to create a PR on it, and then I will know.

**Comment by matsduf** [comment]
Resolved in #269
